### PR TITLE
Add landing page for returning teams

### LINKED ISSUE #1679: Create a landing page for returning teams
This would be linked on the main page instead of just linking to the changelog.

Possible links to include:
Installing SW
Imaging roboRIO
Importing a project
Changelog
Known Issues

### PR DESCRIPTION
**Pull request by Daltz333**
Close #1679

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -26,29 +28,14 @@ Community translations can be found in a variety of languages in the bottom-left
...
-
-         .. div:: col-sm pl-1 pr-1 pt-1
-
-            .. link-button:: /docs/yearly-overview/known-issues
-               :type: ref
-               :text: Known Issues
-               :classes: btn-outline-primary btn-block
+   .. link-button:: /stubs/returning-teams-stub
+      :type: ref
+      :text: Popular Articles
```
Comment (sciencewhiz) [inline source/index.rst:37]: I don't like the name popular articles as it implies good things, not necessary things

File: source/stubs/returning-teams-stub.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+:orphan:
+
+Returning Teams
+===============
+
+This section serves as a launch point for various popular articles that a veteran team may want. It is advised that **all** teams read through the :doc:`changelog </docs/yearly-overview/yearly-changelog>` and :doc:`known issues </docs/yearly-overview/known-issues>` for the season.
+
+.. mimictoc::
+   :maxdepth: 2
```
Comment (sciencewhiz) [inline source/stubs/returning-teams-stub.rst:9]: Is there a way to not expand the individual articles? expanding the imaging articles to headings doesn't make sense

File: source/stubs/returning-teams-stub.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+   /docs/zero-to-robot/step-2/index
+   /docs/zero-to-robot/step-3/imaging-your-roborio
+   /docs/software/roborio-info/roborio2-imaging
```
Comment (sciencewhiz) [inline source/stubs/returning-teams-stub.rst:13]: add programming radio

**Review by sciencewhiz — COMMENTED** [review]
Rather then an index, I think it would be better as an article, that says first install the software, and link to LabVIEW, Game Tools, and WPILib instructions. Then say image your roboRIO and link to the imaging instructions. Then program the radio, then import your old project, etc.

**Comment by Daltz333** [comment]
Yeah, I can do that. A bit more work but easily enough.

**Pull request closed without merging**